tests/test_gui/test_list.py

```diff
@@ -25,11 +25,13 @@
 from .test_base import DummyPID, create_app
 
 dummy_output = """------------------------------
+Public ID: fetchai/default:0.1.0
```

Comment:
the mock output was not updated, and so the CLI GUI in the tests failed to retrieve the public id with `aea list`